bfdd/bfdd_nb_config.c

```diff
@@ -519,18 +700,28 @@ int bfdd_bfd_profile_required_echo_receive_interval_modify(
 	struct nb_cb_modify_args *args)
 {
 	struct bfd_profile *bp;
+	uint32_t min_rx;
 
 	switch (args->event) {
 	case NB_EV_VALIDATE:
+		min_rx = yang_dnode_get_uint32(args->dnode, NULL);
+		if (min_rx == 0)
+			return NB_OK;
+		if (min_rx < 10000 || min_rx > 60000000)
+			return NB_ERR_VALIDATION;
 		break;
```

Comment:
ditto : yang change
separate commit for changes related to common bfd.

Reply:
you seem to have removed those changes.
if the behavior can be kept unchanged, it is ok to me.